packages/dumb/src/models/focused-slice.model.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2023, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+export interface FocusedSliceData {
+  firstQuartile: number
+  max: number
+  median: number
+  min: number
+  ninetiethQuantile: number
+  tenthQuantile: number
+  thirdQuartile: number
+  msFrom: number
+  msTo: number
+}
+
+export interface FocusedSlice {
+  data: FocusedSliceData
+  position: {
+    left: number
+    tooltipLeft: boolean
+    yPositions: {
+      firstQuartile: number
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏We could create an interface for the common properties between `FocusedSliceData` and `FocusedSlice.position.yPositions`?

The interface could be named `SliceKeyIndicators`
```ts
interface SliceKeyIndicators {
  firstQuartile: number
  max: number
  median: number
  min: number
  ninetiethQuantile: number
  tenthQuantile: number
  thirdQuartile: number
}

export interface FocusedSliceData extends SliceKeyIndicators {
  msFrom: number
  msTo: number
}

export interface FocusedSlice {
  data: FocusedSliceData
  position: {
    left: number
    tooltipLeft: boolean
    yPositions: SliceKeyIndicators
  }
}
```